addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -91,7 +91,7 @@
                 <group>
                     <group>
                         <field name="name" attrs="{'readonly':[('state','!=','draft'),('state','!=','confirm')]}"/>
-                        <field name="holiday_status_id" context="{'employee_id':employee_id}"/>
+                        <field name="holiday_status_id" context="{'employee_id':employee_id, 'leave_date':date_from}"/>
```

Comment:
To check, but I think that we could rename it to default_date_from, meaning it will be correctly managed if we update compute_valid and search_valid as we won't have to override default_get in leave anymore.